libtransact/src/workload/batch_gen.rs

```diff
@@ -25,7 +25,7 @@ use cylinder::Signer;
 use protobuf::Message;
 
 use crate::error::{InternalError, InvalidStateError};
-use crate::protocol::batch::{BatchBuilder, BatchPair};
+use crate::protocol::batch::{Batch as ProtocolBatch, BatchBuilder, BatchPair};
```

Comment:
I think it would be good here to use Batch/Transaction from protocol and rename the protos batch instead (ProtobufBatch?).

Reply:
Fixed these to use protocol Batch/Transaction and rename protos Batch/Transaction to ProtobufBatch/ProtobufTransaction